pkgs/servers/x11/xorg/overrides.nix

```diff
@@ -98,243 +99,247 @@ in
     # prevents "misaligned_stack_error_entering_dyld_stub_binder"
     configureFlags = lib.optionals isDarwin [
       "CFLAGS=-O0"
     ];
-  };
+  });
 
-  libXxf86vm = attrs: attrs // {
+  libXxf86vm = super.libXxf86vm.overrideAttrs (attrs: {
     outputs = [ "out" "dev" ];
     configureFlags = attrs.configureFlags or []
       ++ malloc0ReturnsNullCrossFlag;
-  };
+  });
 
   # Propagate some build inputs because of header file dependencies.
   # Note: most of these are in Requires.private, so maybe builder.sh
   # should propagate them automatically.
-  libXt = attrs: attrs // {
+  libXt = super.libXt.overrideAttrs (attrs: {
     preConfigure = ''
       sed 's,^as_dummy.*,as_dummy="\$PATH",' -i configure
     '';
     configureFlags = attrs.configureFlags or []
       ++ malloc0ReturnsNullCrossFlag;
-    propagatedBuildInputs = [ xorg.libSM ];
+    propagatedBuildInputs = [ self.libSM ];
     CPP = stdenv.lib.optionalString stdenv.isDarwin "clang -E -";
     outputs = [ "out" "dev" "devdoc" ];
-  };
+  });
 
   # See https://bugs.freedesktop.org/show_bug.cgi?id=47792
   # Once the bug is fixed upstream, this can be removed.
-  luit = attrs: attrs // {
+  luit = super.luit.overrideAttrs (attrs: {
     configureFlags = [ "--disable-selective-werror" ];
-  };
+  });
 
-  compositeproto = attrs: attrs // {
-    propagatedBuildInputs = [ xorg.fixesproto ];
-  };
+  compositeproto = super.compositeproto.overrideAttrs (attrs: {
+    propagatedBuildInputs = [ self.fixesproto ];
+  });
 
-  libICE = attrs: attrs // {
+  libICE = super.libICE.overrideAttrs (attrs: {
     outputs = [ "out" "dev" "doc" ];
-  };
+  });
 
-  libXcomposite = attrs: attrs // {
+  libXcomposite = super.libXcomposite.overrideAttrs (attrs: {
     outputs = [ "out" "dev" ];
-    propagatedBuildInputs = [ xorg.libXfixes ];
-  };
+    propagatedBuildInputs = [ self.libXfixes ];
+  });
 
-  libXaw = attrs: attrs // {
+  libXaw = super.libXaw.overrideAttrs (attrs: {
     outputs = [ "out" "dev" "devdoc" ];
-    propagatedBuildInputs = [ xorg.libXmu ];
-  };
+    propagatedBuildInputs = [ self.libXmu ];
+  });
 
-  libXcursor = attrs: attrs // {
+  libXcursor = super.libXcursor.overrideAttrs (attrs: {
     outputs = [ "out" "dev" ];
-  };
+  });
 
-  libXdamage = attrs: attrs // {
+  libXdamage = super.libXdamage.overrideAttrs (attrs: {
     outputs = [ "out" "dev" ];
-  };
+  });
 
-  libXft = attrs: attrs // {
+  libXft = super.libXft.overrideAttrs (attrs: {
     outputs = [ "out" "dev" ];
-    propagatedBuildInputs = [ xorg.libXrender args.freetype args.fontconfig ];
+    propagatedBuildInputs = [ self.libXrender args.freetype args.fontconfig ];
     configureFlags = attrs.configureFlags or []
       ++ malloc0ReturnsNullCrossFlag;
     # the include files need ft2build.h, and Requires.private isn't enough for us
     postInstall = ''
       sed "/^Requires:/s/$/, freetype2/" -i "$dev/lib/pkgconfig/xft.pc"
     '';
-  };
+    passthru = {
```

Comment:
The generated file no longer applies `// { inherit $buildInputs; }` to each package. Qt3 relies on being able to access `libXft.freetype` and `libXft.fontconfig`. Updated to `passthru`.